Loop/View Controllers/CarbEntryViewController.swift

```diff
@@ -493,11 +495,13 @@ extension CarbEntryViewController: EmojiInputControllerDelegate {
             return
         }
 
-        let lastAbsorptionTime = self.absorptionTime
-        self.absorptionTime = orderedAbsorptionTimes[section]
-
-        if let cell = tableView.cellForRow(at: IndexPath(row: Row.absorptionTime.rawValue, section: 0)) as? DateAndDurationTableViewCell {
-            cell.duration = max(lastAbsorptionTime ?? 0, orderedAbsorptionTimes[section])
+        if foodType == nil || foodType!.isEmpty {
```

Comment:
What will happen if the user taps a fast absorption emoji, then deletes it, then taps a slow absorption emoji? This logic will change it for both emoji. Is that desired here?

Reply:
Correct. I understand that the desired behaviour is that only the first entered emoji updates the absorption time.